tests/integration/redis/test_basic.php

```diff
@@ -88,27 +195,83 @@ function test_basic() {
   $redis->connect($REDIS_HOST, $REDIS_PORT);
 
   /* generate a unique key to use for this test run */
-  $key = randstr(16);
-  if ($redis->exists($key)) {
-    echo "key already exists: ${key}\n";
+  $key1 = randstr(16);
+  $key2 = randstr(16);
+  if ($redis->exists([$key1, $key2])) {
+    echo "key(s) already exist: ${key1}, ${key2}\n";
```

Comment:
Maybe this could be a SKIP situation since the test was not run?

Reply:
Perhaps, but I think in that case every other test would need to be changed too as a similar pattern is in every one of them.

Reply:
Happy to add this.  To clarify, the test subsystem just looks for the `skip:` literal?

As in, this would suffice?
```php
  if ($redis->exists([$key1, $key2])) {
    die("skip: key(s) already exist: ${key1}, ${key2}\n");
  }
  ```
For consistency, I will change this block in every test file so I wanted to make sure it's correct.

Reply:
@michael-grunder Yes, this is sufficient.

Reply:
Resolved via [8571000](https://github.com/newrelic/newrelic-php-agent/pull/481/commits/8571000f04b6edf936be83e3cee2ece7ea7dbd94)